tensor.go

```diff
@@ -385,6 +413,11 @@ func Eq(a, other Tensor) Tensor {
 	return Tensor{(*unsafe.Pointer)(&t)}
 }
 
+// Eq torch.eq
+func (a Tensor) Eq(other Tensor) Tensor {
```

Comment:
What is the difference between Eq and Equal?

Reply:
[Eq](https://pytorch.org/docs/stable/generated/torch.eq.html) computes element-wise equality, the return value is a bool Tensor.

[Equal])(https://pytorch.org/docs/stable/generated/torch.equal.html) compares two tensors, the return value is a bool value.